platformio.ini

```diff
@@ -74,3 +95,21 @@ monitor_port = /dev/ttyUSB0
 build_flags = '-w'
               '-D BLUEPILL'
               '-D UWB_ANCHOR'
+
+[env:bp_anchor_ad]
+# extends doesn't work, great job platformio. That means
+# we have to duplicate all the environment lines.
+# TODO(michalc): figure out why extends doesn't work
+extends = bp_tag
+platform = ststm32
+board = bluepill_f103c8
+framework = arduino
+#lib_deps = thotro/DW1000@^0.9
+#upload_flags = -c set CPUTAPID 0x2ba01477
+upload_protocol = stlink
+monitor_speed = 115200
+monitor_port = /dev/ttyUSB0
+build_flags = '-w'
+              '-D BLUEPILL'
+              '-D UWB_ANCHOR'
+			  '-D UWB_ANCHOR_AD'
```

Comment:
something about your editor setup is wonky with those weird indentations.

Reply:
wtf :disappointed:

Reply:
Yeah I dont know, look good in my editor.

Reply:
oh well, no biggie, can be fixed later. Approving. I'll push the `.editorconfig` after this. If you have the editorconfig plugin installed then it should solve it.